computable/tests/reserve_test.py

```diff
@@ -9,110 +9,110 @@ def test_deploy(reserve):
     assert reserve.account != reserve.address
```

Comment:
The spacing in this file was weird (3-spaces instead of the usual 4) so fixed that